ServiceDesk.Abstractions/IServiceDeskClient.cs

```diff
@@ -1,16 +1,17 @@
-﻿namespace ServiceDesk.Abstractions
+﻿namespace ArcadiaTeamsBot.ServiceDesk.Abstractions
 {
-    using ServiceDesk.Abstractions.DTOs;
     using System.Collections.Generic;
     using System.Threading;
     using System.Threading.Tasks;
 
+    using ArcadiaTeamsBot.ServiceDesk.Abstractions.DTOs;
+
     public interface IServiceDeskClient
     {
         Task<IEnumerable<ServiceDeskRequestTypeDTO>> GetRequestTypes(CancellationToken cancellationToken);
 
         Task<IEnumerable<ServiceDeskRequestDTO>> GetCurrentRequests(string username, CancellationToken cancellationToken);
 
-        Task<IEnumerable<ServiceDeskRequestPriorityDTO>> GetPriorities(CancellationToken cancellationToken);
```

Comment:
Revert it. It should have usual behavior for all methods and it shouldn't depend on internal implementation.

Reply:
Reverted

Reply:
Reverted